apps/storybook/package.json

```diff
@@ -4,44 +4,35 @@
   "private": true,
   "dependencies": {},
   "devDependencies": {
+    "@fontsource/noto-sans": "5",
+    "@fontsource/noto-sans-mono": "5",
     "@itwin/itwinui-icons-react": "2",
     "@itwin/itwinui-react": "*",
-    "@storybook/addon-a11y": "~7.0.2",
-    "@storybook/addon-actions": "~7.0.2",
-    "@storybook/addon-essentials": "~7.0.2",
-    "@storybook/addon-mdx-gfm": "~7.0.2",
-    "@storybook/addon-storysource": "~7.0.2",
-    "@storybook/node-logger": "~7.0.2",
-    "@storybook/react": "~7.0.2",
-    "@storybook/react-vite": "~7.0.2",
-    "@types/node": "^16.11.31",
-    "@types/react": "^18.0.2",
-    "@types/react-dom": "^18.0.2",
+    "@ladle/react": "^3.3.1",
+    "@types/node": "*",
+    "@types/react": "*",
+    "@types/react-dom": "*",
     "configs": "*",
     "cypress": "13.2.0",
     "cypress-image-diff-js": "1.30.1",
     "dotenv-cli": "7.0.0",
-    "eslint": "^8.14.0",
+    "eslint": "*",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-storybook": "^0.6.11",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
-    "react-table": "^7.1.0",
-    "serve": "^14.0.1",
-    "sb": "~7.0.2",
-    "storybook-dark-mode": "^3.0.0",
-    "typescript": "~5.1.6",
-    "vite": "^4.1.5"
+    "typescript": "*",
```

Comment:
Is there a benefit for having `*` as the version for multiple packages?

I remember you once said that typescript sometimes has breaking changes even in minor releases. So with `*`, do we automatically use the latest version and thus might need to often make modifications to our code?

Reply:
`*` basically means "anything is fine" rather than "i want latest". i did it this way to help avoid unnecessary duplicate versions as we use some of the same deps in multiple places in the monorepo

the actual typescript version is controlled in the root package.json, so this should be fine fine